docs/DeveloperGuide.md

```diff
@@ -382,9 +389,9 @@ Given below is an example usage scenario of the `SearchAnimalCommand`:
 2. The user executes the `search` command with the prefixes of the attributes to be searched. For instance, `search b/Poodle` would search for animals of breed Poodle.
 3. The command verifies the validity of the prefixes. If all the prefixes are invalid, it throws a `CommandException`. Otherwise, it retrieves the animals that match the valid prefixes, and returns a successful command result.
 
-The following sequence diagram shows how the `SearchAnimalCommand` works:
+> [!NOTE]
+> Refer to `ListAnimalCommand` for the sequence diagrams of the model component of `SearchAnimalCommand` as they are largely similar, except for the predicate passed into updateFilteredAnimalList.
```

Comment:
Possible typo here! do you mean for the sequence diagrams **and** the model component?